usrsctplib/netinet/sctp_pcb.c

```diff
@@ -5293,7 +5293,7 @@ sctp_add_vtag_to_timewait(uint32_t tag, uint16_t lport, uint16_t rport)
 
 	SCTP_INP_INFO_WLOCK_ASSERT();
 	(void)SCTP_GETTIME_TIMEVAL(&now);
-	time = now.tv_sec + SCTP_BASE_SYSCTL(sctp_vtag_time_wait);
+	time = (uint32_t)(now.tv_sec + SCTP_BASE_SYSCTL(sctp_vtag_time_wait));
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/64707067d0c75c0a7aab5ad5d4d00611988e2a7d